data_model/src/account.rs

```diff
@@ -163,22 +163,6 @@ pub mod model {
     pub struct SignatureCheckCondition(pub EvaluatesTo<bool>);
 }
 
-impl AccountId {
-    #[cfg(feature = "transparent_api")]
-    const GENESIS_ACCOUNT_NAME: &str = "genesis";
-
-    /// Construct [`Id`] of the genesis account.
-    #[inline]
-    #[must_use]
-    #[cfg(feature = "transparent_api")]
-    pub fn genesis() -> Self {
```

Comment:
How can we construct genesis account now?

Reply:
it's now a constant `GENESIS_ACCOUNT` and is stored in `iroha_genesis`